api/types/database.go

```diff
@@ -680,6 +693,16 @@ func (gcp GCPCloudSQL) GetServerName() string {
 	return fmt.Sprintf("%s:%s", gcp.ProjectID, gcp.InstanceID)
 }
 
+// HasValidID returns true if any of the identifiers exists.
+func (rds RDS) HasValidID() bool {
+	return rds.ClusterID != "" || rds.InstanceID != "" || rds.ProxyName != "" || rds.ProxyCustomEndpointName != ""
+}
+
+// IsRDSProxy returns true if this is an RDS Proxy.
+func (rds RDS) IsRDSProxy() bool {
```

Comment:
nit: right now we have: 
`RDS.IsRDSProxy() `
what would you say about just: 
`RDS.IsProxy()`
```suggestion
func (rds RDS) IsProxy() bool {
```